IceCubesApp/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -130,6 +130,10 @@
 "settings.content.media.show.alt" = "Show ALT Texts";
 "settings.content.reading" = "Reading";
 "settings.content.posting" = "Posting";
+"settings.content.sharing" = "Sharing";
```

Comment:
Those strings need to be added to every localization files. You can leave them in English as it'll be translated later. Thanks!

Reply:
addressed in b95741b